doc/api/videoThumbnailLoader.md

```diff
@@ -0,0 +1,131 @@
+and easier to decode, they are preferred over standard video tracks for creating
+thumbnails.
+
+## How to use it ###############################################################
+
+
+As an experimental tool, the VideoThumbnailLoader won't be included in a
+default RxPlayer build.
+
+Instead, it should be imported by adding the RxPlayer through a dependency
+trough the npm registry (e.g. by doing something like ``npm install
+rx-player``) and then specifically importing this tool from
+``"rx-player/experimental/tools"``:
+
+```js
+import { VideoThumbnailLoader } from "rx-player/experimental/tools";
+import RxPlayer from "rx-player";
+
+const player = new RxPlayer({ /* some options */ });
+player.loadVideo({ /* some other options */ });
+
+// Video element used to display thumbnails.
+const thumbnailVideoElement = document.createElement("video");
+const videoThumbnailLoader = new VideoThumbnailLoader(
+  thumbnailVideoElement,
+  player
+);
+
+videoThumbnailLoader.setTime(200);
+```
+
+## Functions ###################################################################
+
+### addFetcher #################################################################
+
+_arguments_:
+  - _fetcher_ (``Object``): Imported fetcher from VideoThumbnailLoader package.
+
+To be able to load and parse segments from a specific streaming format, you may
+import the corresponding fetcher from the VTL package, and add it to the
+instance through this function.
+
+#### Example
+
+```js
+  import {
+    DASH_FETCHER, /* dash fetcher */
+    SMOOTH_FETCHER, /* smooth fetcher */
+    MPL_FETCHER, /* metaplaylist fetcher */
+  } from "rx-player/experimental/tools/videoThumbnailLoader";
+  videoThumbnailLoader.addFetcher(DASH_FETCHER);
+  videoThumbnailLoader.addFetcher(SMOOTH_FETCHER);
+  videoThumbnailLoader.addFetcher(MPL_FETCHER);
+```
+
+### setTime ####################################################################
+
+_arguments_:
+
+  - _time_ (``number``): Time for which we want to display a thumbnail.
+
+_return value_: ``Promise``
+
+From a given time, load video segments, and append to video element.
+The loader find the period for the given time. In this period, it chooses
+the first video adaptation. If a trick mode track is present on it, choose it.
+Otherwise, select the first representation.
+
+#### Return value
+
+The return value is a Promise.
+It :
+- ``resolve`` when the thumbnail for given time has been loaded.
+- ``reject`` in case of error : return an error.
+
+The promise does not only rejects when setting thumbnail has failed. There are
+some cases where the thumbnail loader decides not to load. Here is a list of
+every failure code (``error.code``) :
+- NO_MANIFEST : No manifest available on current RxPlayer instance.
+- NO_TRACK : In the player manifest, there are either no period or no
+             representation to get video chunks.
+- MISSING_INIT_DATA : The chosen track does not have an init data, so it can't
+                      be buffered.
```

Comment:
I don't get this one. Technically we could have no initialization segment and still be able to play (if the corresponding init data is integrated in all media segments).